dashboard/config/marketing/announcements.json

```diff
@@ -29,15 +29,6 @@
       "buttonUrl": "https://code.org/music",
       "buttonId": "music-lab-launch"
     },
-    "lms-launch": {
```

Comment:
This was already replaced by the Curriculum Launch banner which is currently hidden behind the `curriculum-launch-2024` DCDO flag.